packages/backend/src/graphql/mutations/create-flow-transfer.ts

```diff
@@ -0,0 +1,27 @@
+import FlowTransfer from '@/models/flow-transfers'
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    flowId: string
+    newOwnerId: string
+  }
+}
+
+const createFlowTransfer = async (
+  _parent: unknown,
+  params: Params,
+  context: Context,
+) => {
+  const { flowId, newOwnerId } = params.input
+  const transfer: FlowTransfer = await context.currentUser
+    .$relatedQuery('sentFlowTransfers')
+    .insert({
```

Comment:
is there anything blocking the user from transferring the same flow twice? there is a chance the mutation may be sent more than once (e.g. user clicks the button multiple times by accident / a network proxy along the path retries the request)

Reply:
nvm I see this is handled in #472